docs/portal/image-registry/adding-projects.md

```diff
@@ -1,17 +1,19 @@
 # Adding Projects
 
 Admin portal users can use the following procedure to add projects.
+
 **Note:** Non-admin portal users are not permitted to add projects.
+
 1. In the left pane, click **Image Registry > Registry Projects**.
 2. At the top right of the Registry Projects page, click the **\+ Add New Project** button. 
 3. Complete the fields in the Add Project form. Required fields are denoted by an asterisk (\*).
 
-![null](</docs/resources/images/registry/add-project.png>)
+<p align=center><img src="/docs/resources/images/registry/add-project.png" width="600"></p>
```

Comment:
In previous documentations, screenshots are aligned to the left, and there isn't explicit image width specification. We'd better do the same here.